src/psygnal/_signal.py

```diff
@@ -1127,6 +1127,10 @@ def resume(self, reducer: ReducerFunc | None = None, initial: Any = _NULL) -> No
         # EventedModel.update, it may be undefined (as seen in tests)
         if not getattr(self, "_args_queue", None):
             return
+        if len(self._slots) == 0:
+            self._args_queue = []
```

Comment:
```suggestion
            self._args_queue.clear()
```

don't think object permanence really matters here, but we do `clear()` below, so might as well remain consistent here